release-engineering/role-handbooks/branch-manager.md

```diff
@@ -292,77 +310,143 @@ The following are some ways to determine if the release process was successful:
 
 1. The build tag and release artifacts become [visible on GitHub at https://github.com/kubernetes/kubernetes/releases](https://github.com/kubernetes/kubernetes/releases)
 
-1. The release is logged automatically by [k8s-release-robot](https://github.com/k8s-release-robot) in [sig-release](https://github.com/k8s-release-robot/sig-release)
+2. The release is logged automatically by [k8s-release-robot](https://github.com/k8s-release-robot) in [sig-release](https://github.com/k8s-release-robot/sig-release)
 
-1. CHANGELOG-X.Y.md is automatically loaded into the kubernetes/kubernetes repo: [https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.12.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.12.md)
+3. CHANGELOG-X.Y.md is automatically loaded into the kubernetes/kubernetes repo: [https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.12.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.12.md)
 
-# Create CI/Presubmit jobs
+---
 
-This takes place around week 6-7, as soon as the new `release-X.Y` branch is created. This task is taken over from test-infra when the role was dissolved starting `v1.16`:
+## Create CI/Presubmit jobs
 
-1. `git clone` [`git@github.com:kubernetes/test-infra.git`](https://github.com/kubernetes/test-infra)
+This takes place around week 6-7, as soon as the new [`release-x.y` branch is created](#branch-creation).
 
-1. [Install Bazel](https://docs.bazel.build/versions/master/install.html)
+1. Clone the [test-infra repository](https://github.com/kubernetes/test-infra) e.g., `git clone git@github.com:kubernetes/test-infra.git`
 
-1. Update [`images/kubekins-e2e/variants.yaml`](https://github.com/kubernetes/test-infra/blob/master/images/kubekins-e2e/variants.yaml) by deleting the oldest entry and copying the `master` block to create an entry using the latest version number. For example:
+2. [Install Bazel](https://docs.bazel.build/versions/master/install.html)
+
+3. Update [`images/kubekins-e2e/variants.yaml`](https://github.com/kubernetes/test-infra/blob/master/images/kubekins-e2e/variants.yaml). ([example PR](https://github.com/kubernetes/test-infra/pull/13876))
+
+   (The `variants.yaml` allows us to target various branches or branch combinations during CI tests. The `K8S_VERSION` variable maps to the version marker files viewable in the [`kubernetes-release` GCS bucket](https://gcsweb.k8s.io/gcs/kubernetes-release/release/) e.g., [`latest-1.16.txt`](https://storage.googleapis.com/kubernetes-release/release/latest-1.16.txt))
```

Comment:
This is good, but it's now a great summary buried where only a subset of relevant folks will see it.  IMO should be in https://github.com/kubernetes/test-infra/blob/master/docs/kubernetes-versions.md